ui/job-view/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render } from 'react-dom';
+import { AppContainer } from 'react-hot-loader';
+
+// Vendor Styles
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'font-awesome/css/font-awesome.css';
+
+// Vendor JS
+import 'bootstrap';
+import 'jquery.scrollto';
+
+// Treeherder Styles
+import '../css/treeherder.css';
+import '../css/treeherder-global.css';
+import '../css/treeherder-navbar.css';
+import '../css/treeherder-navbar-panels.css';
+import '../css/treeherder-notifications.css';
+import '../css/treeherder-details-panel.css';
+import '../css/treeherder-job-buttons.css';
+import '../css/treeherder-resultsets.css';
+import '../css/treeherder-pinboard.css';
+import '../css/treeherder-bugfiler.css';
+import '../css/treeherder-loading-overlay.css';
+
+import App from './App';
+
+const load = () => render((
+  <AppContainer>
+    <App />
```

Comment:
Can you pass in `JobView` since it seems the only thing `App` is doing is returning `JobView`?